packages/core/data-transfer/src/strapi/remote/routes.ts

```diff
@@ -31,6 +31,11 @@ export const registerAdminTransferRoute = (strapi: Strapi.Strapi) => {
     method: 'GET',
     path: TRANSFER_URL,
     handler: createTransferHandler(),
-    config: { auth: false },
+    config: {
+      policies: [
+        'admin::isAuthenticatedAdmin',
+        { name: 'admin::hasPermissions', config: { actions: ['admin::transfer.push'] } },
```

Comment:
This PR is implementing the tasks we have for "CLI transfer --to" and "add permissions in strapi admin" and "transfer auth" so this simple check is good enough.

Once I start on "CLI transfer --from" I will have to solve making this an OR, which might require adding or modifying a policy.